osbs/build/build_request.py

```diff
@@ -867,7 +889,13 @@ def render(self, validate=True):
                            # add the path to the plugin's
                            # configuration. This is done elsewhere.
                            None):
-                          self.spec.registry_secrets.value})
+                          self.spec.registry_secrets.value,
+
+                          ('buildstep_plugins', 'orchestrate_build', 'osbs_client_config'):
```

Comment:
What would put the symlink there? It would have to be baked into the builder image? In that case there would need to be a "well-known" mount path for the osbs.conf secret (and a separate one for the dockpulp config if we use this method for the Pulp integration).

We could set a default, sure... but isn't that more work than just making it configurable (which it would have to be anyway)?